sql/updates/world/3.3.5/2040_99_99_99_world.sql

```diff
@@ -0,0 +1,7 @@
+-- Phantom Mammoth 
+DELETE FROM creature WHERE id=29748; -- shouldn't stand all the time
+UPDATE creature_template SET `flags_extra` = `flags_extra`&0x2, `unit_flags` = `unit_flags`|0x00020000 WHERE entry=29748; -- set visible to players, npc shouldn't attack

```

Comment:
missing ` around fields, e.g. creature_template should be -> `creature_template`